op-acceptance-tests/tests/interop/reorgs/invalid_exec_msg_test.go

```diff
@@ -0,0 +1,292 @@
+package reorgs
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/bindings"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/devtest"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+	"github.com/ethereum-optimism/optimism/op-service/txintent"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/seqtypes"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReorgInvalidExecMsg(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	ctx := t.Ctx()
+
+	sys := SimpleInterop(t)
+	l := sys.Log
+
+	ia := sys.Sequencer.IndividualAPI(sys.L2ChainA.ChainID())
+
+	// three EOAs for triggering the init and exec interop txs, as well as a simple transfer tx
+	var alice, bob, cathrine *dsl.EOA
+	{
+		// alice is on chain A
+		pk, err := crypto.GenerateKey()
+		require.NoError(t, err)
+		alice = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELA)
+		sys.FaucetA.Fund(alice.Address(), eth.ThousandEther)
+
+		// bob is on chain B
+		pk, err = crypto.GenerateKey()
+		require.NoError(t, err)
+		bob = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELB)
+		sys.FaucetB.Fund(bob.Address(), eth.ThousandEther)
+
+		// cathrine is on chain A
+		pk, err = crypto.GenerateKey()
+		require.NoError(t, err)
+		cathrine = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELA)
+		sys.FaucetA.Fund(cathrine.Address(), eth.ThousandEther)
```

Comment:
```suggestion
		alice = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELA)
		sys.FaucetA.Fund(alice.Address(), eth.OneEther)

		// bob is on chain B
		pk, err = crypto.GenerateKey()
		require.NoError(t, err)
		bob = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELB)
		sys.FaucetB.Fund(bob.Address(), eth.OneEther)

		// cathrine is on chain A
		pk, err = crypto.GenerateKey()
		require.NoError(t, err)
		cathrine = dsl.NewEOA(dsl.NewKey(t, pk), sys.L2ELA)
		sys.FaucetA.Fund(cathrine.Address(), eth.OneEther)
```

To make the tests not use up as much test funding when running on a more real test network